src/aosm/azext_aosm/configuration_models/onboarding_vnf_input_config.py

```diff
@@ -20,28 +20,28 @@ class VhdImageConfig:
     """Configuration for a VHD image."""
 
     artifact_name: str = field(
-        default="", metadata={"comment": "Optional. Name of the artifact."}
+        default="", metadata={"comment": "Optional. Name of the artifact. Name will be generated if not supplied."}
     )
     version: str = field(
-        default="", metadata={"comment": "Version of the artifact in A-B-C format."}
+        default="", metadata={"comment": "Version of the artifact in A-B-C format. Note the '-' (dash) not '.' (dot)."}
     )
     file_path: str = field(
         default="",
         metadata={
             "comment": (
-                "Optional. File path of the artifact you wish to upload from your local disk. "
-                "Delete if not required.\nRelative paths are relative to the configuration file. "
-                "On Windows escape any backslash with another backslash."
+                "Supply either file_path or blob_sas_url, not both.\n
+                "File path (absolute or relative to this configuration file) of the artifact you wish to upload from your local disk.\n"
+                "Delete if not required. Use Linux slash (/) file separator even if running on Windows."
             )
         },
     )
     blob_sas_url: str = field(
         default="",
         metadata={
             "comment": (
-                "Optional. SAS URL of the blob artifact you wish to copy to your Artifact Store.\n"
+                "Supply either file_path or blob_sas_url, not both.\nSAS URL of the blob artifact you wish to copy to your Artifact Store.\n"
                 "Delete if not required. "

```

Comment:
I think we should replace "Delete if not required" with "Leave blank if not required", because that's what our validation looks for

Reply:
Thanks. Went for "Leave as empty string".